front/src/reducers/osrdconf/index.ts

```diff
@@ -313,7 +313,7 @@ export default function reducer(inputState: OsrdMultiConfState | undefined, acti
         draft.stdcmConf.standardStdcmAllowance = action.standardStdcmAllowance;
         break;
       case UPDATE_POWER_RESTRICTION:
-        draft[section].powerRestriction = action.powerRestriction;
+        draft[section].powerRestrictionRanges = action.powerRestriction;
```

Comment:
Why not using `powerRestrictionRanges` too for `action` ?

Reply:
Done